static/images/favicon/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/images/mstile-150x150.png"/>
+            <TileColor>#da532c</TileColor>
```

Comment:
This is an orangey color, I don't think it's going to look well with the icon?
<img width="563" alt="Screenshot 2022-01-18 at 12 48 38" src="https://user-images.githubusercontent.com/1009343/149970746-b824c90e-8018-449e-80b6-a839dd424e32.png">

Reply:
It's a job for the PM :) mais i can change it.

Reply:
With the orange

![Uploading Capture d’écran de 2022-01-19 09-11-39.png…]()

Reply:
With the orange 

![Capture d’écran de 2022-01-19 09-11-39](https://user-images.githubusercontent.com/5064590/150090975-20bfe356-416d-4deb-bdbd-95386b75dfc8.png)